packages/document/main-doc/docs/en/guides/basic-features/html.mdx

```diff
@@ -24,44 +24,44 @@ For example the following directory structure:
     └── modern-app-env.d.ts
 ```
 
-`entry-a` will take precedence over the `Docoument.[jt]sx` file under the current entry. If the current entry does not have a `Document.[jt]sx` file, such as `entry-b`, it will look for the `Document.[jt]sx` file in the root directory.
+`entry-a` will first use the `Docoument.[jt]sx` file under the current entry. If there is no `Document.[jt]sx` file under the current entry, like `entry-b`, it will look for the `Document.[jt]sx` file under the root directory.
 
-If not, it will go to the the logic of traditional templates.
+If not found, Modern.js default template will be used.
 
-### HTML Component
+### HTML Components
 
-Modern.js provides a list of components for rendering pages to help developers generate templates, which can be exported from `@modern-js/runtime/document`:
+Modern.js provides a series of components for rendering pages to help developers generate templates. These components can be used from `@modern-js/runtime/document`:
 
 ```tsx
 import { Html, Body, Root, Head, Scripts } from '@modern-js/runtime/document';
 ```
 
-These components are rendered:
+These components are used:
 
-- `Html`: Provide the ability of native HTML Elements, and render necessary components that the developer did not add by default. `<Head>` and `<Body>` must exist, and other components can be assembled by selecting appropriate components on demand.
+- `Html`: provides the ability of native HTML Element and can render the necessary components that developers have not added by default. `<Head>` and `<Body>` must exist, and other components can be selected and assembled as needed.
```

Comment:
There's a syntax issue: provides -> Provide.

The original version is more natural.

Reply:
provide the ability of native HTML Element and and render necessary components that the developer did not add by default. `<Head>` and `<Body>` must exist, and other components can be assembled as needed.